src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -1,6 +1,11 @@
 {% extends "richie/fullwidth.html" %}
 {% load cms_tags i18n extra_tags pagination_tags static thumbnail humanize %}
 
+{% block head_title %}
+    {% page_attribute "page_title" as course_title %}
+    {% blocktrans with course_title=course_title %}{{ course_title }} - Course{% endblocktrans %}
```

Comment:
As @sampaccoud mentionned in its comment, I think you could apply this to program and category page.

Reply:
Alright! Noting this for the pull requests related to those pages